libs/api/database/src/lib/content/schemas/tags.schema.ts

```diff
@@ -28,3 +28,4 @@ export class TagsDocument extends Document implements TagsModel {
 }
 
 export const TagsSchema = SchemaFactory.createForClass(TagsDocument);
+TagsSchema.index({ parent: 1, name: 1 })
```

Comment:
nit: I almost want to say that we should have a comment mentioning this index up at the top of the file, because a) this is the first time we've used a compound index and b) indices usually get declared inline, so this being down here will be a bit of a surprise.